packages/hardhat-chai-matchers/src/internal/reverted/revertedWith.ts

```diff
@@ -1,45 +1,53 @@
+import { buildAssert } from "../../utils";
 import { decodeReturnData, getReturnDataFromError } from "./utils";
 
 export function supportRevertedWith(Assertion: Chai.AssertionStatic) {
   Assertion.addMethod(
     "revertedWith",
     function (this: any, expectedReason: unknown) {
+      // see buildAssert's jsdoc
```

Comment:
I'd like to see a one-line explanation here, something like `capture negated flag before async code executes. see buildAssert's jsdoc.`